apps/doji-backend/src/report/report.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { CurrentUser, UserAuthGuard } from '@backend/auth/user.guard'
+import { User } from '@backend/entities/User'
+import { ReportDTO } from '@backend/report/report.dto'
+import { ReportService } from '@backend/report/report.service'
+import { Body, Controller, Post, UseGuards } from '@nestjs/common'
+import { ApiCookieAuth } from '@nestjs/swagger'
+
+@Controller('report')
```

Comment:
not sure ถ้า report มันคือการ report service มันควรอยู่ใน `service` รึเปล่า?

Reply:
report expert